### PR TITLE
xplr: add page

### PR DESCRIPTION
**Pull request by sayanarijit**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/xplr.md
Comment (bl-ue) [inline pages/common/xplr.md:8]:
```suggestion
`xplr {{path/to/directory}}`
```

File: pages/common/xplr.md
Comment (bl-ue) [inline pages/common/xplr.md:12]:
```suggestion
`xplr {{path/to/file}}`
```

File: pages/common/xplr.md
Comment (bl-ue) [inline pages/common/xplr.md:16]: Hmm, this is a cool example, but I'm not sure if it fits our guidelines 🤔

It may though, so let's wait and see.

**Review by bl-ue — COMMENTED** [review]
Thanks for the new page and the cool tool!

File: pages/common/xplr.md
Comment (bl-ue) [reply pages/common/xplr.md:16]: ping @sbrl.  What do you think about this?

File: pages/common/xplr.md
Comment (bl-ue) [inline pages/common/xplr.md:24]: I'm not _totally_ sure about this one because it introduces **2**, not 1 other command, but I doubt that really matters.

**Review by bl-ue — APPROVED** [review]

File: pages/common/xplr.md
Comment (navarroaxel) [reply pages/common/xplr.md:24]: We could base on this comment https://github.com/tldr-pages/tldr/pull/5449#issuecomment-808467937

Comment (sayanarijit) [reply pages/common/xplr.md:24]: So `xplr` as of now doesn't have much command-line functionality. So I think I can remove the `mv` example.
Shall I also remove the `cp` and `vim` examples?

Comment (navarroaxel) [reply pages/common/xplr.md:24]: @sayanarijit just wait for another review.

File: pages/common/xplr.md
Comment (sbrl) [inline pages/common/xplr.md:3]: Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Terminal-based file system explorer.
```

File: pages/common/xplr.md
Comment (sbrl) [reply pages/common/xplr.md:16]: Hrm, I'm not sure. Generally we want to demonstrate the functionality of the command itself, rather than the that of other commands and the builtin shell.

File: pages/common/xplr.md
Comment (sbrl) [reply pages/common/xplr.md:24]: I'd prefer it if we could remove this example. It's demonstrating the functionality of other commands rather than that of the command in question.

**Review by sbrl — APPROVED** [review]
Thanks for the new page, and welcome to tldr-pages, @sayanarijit! I've left some comments below for you to review.

**Comment by sayanarijit** [comment]
Thanks for your review. Removed the examples in question.

**Review by navarroaxel — APPROVED** [review]
Thanks @sayanarijit, please the next time don't push force to the PR, it's harder to track the suggestions and changes.

**Pull request merged**
